compiler/src/dmd/hdrgen.d

```diff
@@ -2161,11 +2160,8 @@ private void expressionPrettyPrint(Expression e, ref OutBuffer buf, ref HdrGenSt
             default:
                 /* This can happen if errors, such as
                  * the type is painted on like in fromConstInitializer().
+                 * Just ignore
                  */
-                if (!global.errors)
-                {
-                    assert(0);
-                }
```

Comment:
this looks to be unrelated, please lave for a separate pr

Reply:
It is related, as it requires the import.